src/components/WinnerPage/Winner.css

```diff
@@ -0,0 +1,325 @@
+              transform: translateY(500px);
+      -webkit-animation-timing-function: ease-in;
+              animation-timing-function: ease-in;
+      opacity: 0;
+    }
+    38% {
+      -webkit-transform: translateY(0);
+              transform: translateY(0);
+      -webkit-animation-timing-function: ease-out;
+              animation-timing-function: ease-out;
+      opacity: 1;
+    }
+    55% {
+      -webkit-transform: translateY(65px);
+              transform: translateY(65px);
+      -webkit-animation-timing-function: ease-in;
+              animation-timing-function: ease-in;
+    }
+    72% {
+      -webkit-transform: translateY(0);
+              transform: translateY(0);
+      -webkit-animation-timing-function: ease-out;
+              animation-timing-function: ease-out;
+    }
+    81% {
+      -webkit-transform: translateY(28px);
+              transform: translateY(28px);
+      -webkit-animation-timing-function: ease-in;
+              animation-timing-function: ease-in;
+    }
+    90% {
+      -webkit-transform: translateY(0);
+              transform: translateY(0);
+      -webkit-animation-timing-function: ease-out;
+              animation-timing-function: ease-out;
+    }
+    95% {
+      -webkit-transform: translateY(8px);
+              transform: translateY(8px);
+      -webkit-animation-timing-function: ease-in;
+              animation-timing-function: ease-in;
+    }
+    100% {
+      -webkit-transform: translateY(0);
+              transform: translateY(0);
+      -webkit-animation-timing-function: ease-out;
+              animation-timing-function: ease-out;
+    }
+   }   
+/* Fin anim return Arrow*/
+body {
+    background-color: rgba(0, 255, 0, .8);
+
+ }
+.winnerResult{
+    display: flex;
+    align-items: flex-start;
+    margin-left: 9%;
+    
+}
+.trophy{
+    width: 61%; 
+    margin: 25px 0 0 0;
+}
+.roll-in-blurred-left {
+    -webkit-animation: roll-in-blurred-left 1.00s cubic-bezier(0.230, 1.000, 0.320, 1.000) both;
+            animation: roll-in-blurred-left 1.00s cubic-bezier(0.230, 1.000, 0.320, 1.000) both;
+}
+.v_1 {
+    font-size: 20px;
+    display: flex;
+    flex-direction: column;
+    margin: 0 0 0 30px;
+    justify-content: space-around;
+}
+.winResultTitle{
+    margin: 0 0 2px 0;
+    font-size: 50px;
+}
+.slide-in-blurred-top {
+    -webkit-animation: slide-in-blurred-top 1s cubic-bezier(0.230, 1.000, 0.320, 1.000) 1s both;
+            animation: slide-in-blurred-top 1s cubic-bezier(0.230, 1.000, 0.320, 1.000) 1s both;
+}
+.winnerPokemon{   
+    width: 80%;
+    margin: 0 0 0 0;
+}
+.rotate-in-center {
+    -webkit-animation: rotate-in-center 0.6s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
+            animation: rotate-in-center 0.6s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
+}
+.option{
+    display: flex;
+    width: 100%;
+    margin: 20px 0 0 0;
+    justify-content: space-around;
+}
+/*Début du code pour le bouton
+button {
```

Comment:
Don't use html selectors in the component's css file.

Reply:
Otherwise you risk impacting other components in your page. This way components wouldn't be independant.